src/docs/sdk/performance/modules/assets.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Assets Module'
+---
+
+The javascript SDK should automatically instrument most assets fetched by the application. This includes images, stylesheets, scripts, and other assets. But if you may wish to instrument assets manually.
```

Comment:
`But if you may wish to instrument assets manually.` I think you missed some words in this sentence